### PR TITLE
mypy & run fixes

### PR DESCRIPTION
**Pull request by wdika**
Edited and fixes #94

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
         "torch.utils.data",
     ]:
         sys.modules[m] = mock.Mock(name=m)
-    sys.modules["torch"].__version__ = "1.5"  # fake version
+    sys.modules["torch"].__version__ = "1.5"  # type: ignore
```
Comment (jonasteuwen) [inline docs/conf.py:49]: I think we need to update this to 1.8.

File: docs/conf.py [hunk 2]
```diff
@@ -177,7 +177,7 @@
 
 # -- Options for LaTeX DoIterationOutput ------------------------------------------
```
Comment (jonasteuwen) [inline docs/conf.py:178]: Can you change this to "output"? It was a wrong replace...

File: docs/conf.py [hunk 3]
```diff
@@ -177,7 +177,7 @@
...
 
-latex_elements = {
+latex_elements: Dict[type, type] = {
```
Comment (jonasteuwen) [inline docs/conf.py:180]: I don't know what this is of type "type", isn't "Dict" good enough?

**Review by jonasteuwen — CHANGES_REQUESTED** [review]
Well done. See requested changes.

File: docs/conf.py (hunk 3 above)
Comment (wdika) [reply docs/conf.py:180]: Yes it is.

**Review by jonasteuwen — CHANGES_REQUESTED** [review]
Please confirm the model runs both train & prediction.

**Comment by wdika** [comment]
> Please confirm the model runs both train & prediction.

After my latest commit model run both train & prediction, and also passes mypy tests & black.

**Review by jonasteuwen — APPROVED** [review]

**Pull request merged**